src/map/spell.h

```diff
@@ -1090,26 +1090,29 @@ class CSpell
     bool               isRaise();          // is a raise spell (e.g. Trust: Ferreous Coffin)
     bool               canHitShadow();     // check if spell ignores shadows
 
-    void setRadius(float radius);
-    void setTotalTargets(uint16 total);
-    void setID(SpellID id);
-    void setJob(int8* jobs);
-    void setMPCost(uint16 MP);
-    void setCastTime(uint32 CastTime);
-    void setRecastTime(uint32 RecastTime);
-    void setValidTarget(uint16 ValidTarget);
-    void setAnimationID(uint16 AnimationID);
-    void setAnimationTime(uint16 AnimationTime);
-    void setSpellGroup(SPELLGROUP SpellGroup);
-    void setSpellFamily(SPELLFAMILY SpellFamily);
-    void setSkillType(uint8 SkillType);
-    void setZoneMisc(uint16 Misc);
-    void setAOE(uint8 AOE);
-    void setBase(uint16 base);
-    void setElement(uint16 element);
-    void setMultiplier(float multiplier);
-    void setMessage(uint16 message);
-    void setMagicBurstMessage(uint16 message);
+    void     setRadius(float radius);
+    void     setTotalTargets(uint16 total);
+    void     setID(SpellID id);
+    void     setJob(int8* jobs);
+    void     setMPCost(uint16 MP);
+    void     setCastTime(uint32 CastTime);
+    void     setRecastTime(uint32 RecastTime);
+    void     setValidTarget(uint16 ValidTarget);
+    void     setAnimationID(uint16 AnimationID);
+    void     setAnimationTime(uint16 AnimationTime);
+    void     setSpellGroup(SPELLGROUP SpellGroup);
+    void     setSpellFamily(SPELLFAMILY SpellFamily);
+    void     setSkillType(uint8 SkillType);
+    void     setZoneMisc(uint16 Misc);
+    void     setAOE(uint8 AOE);
+    void     setBase(uint16 base);
+    void     setElement(uint16 element);
+    void     setMultiplier(float multiplier);
+    void     setMessage(uint16 message);
+    void     setMagicBurstMessage(uint16 message);
+    MODIFIER getModifier();
```

Comment:
No need to shift all all this alignment;

auto getModifier() -> MODIFIER;

Reply:
done